scaffolder-templates/docs-template-cookiecutter/skeleton/catalog-info.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: {{cookiecutter.name | jsonify}}
+  description: {{cookiecutter.description | jsonify}}
+  annotations:
+    github.com/project-slug: {{cookiecutter.destination.owner + "/" + cookiecutter.destination.repo}}
+    backstage.io/techdocs-ref: url:{{"https://" + cookiecutter.destination.host + "/" + cookiecutter.destination.owner + "/" + cookiecutter.destination.repo}}
```

Comment:
Does this make sense to keep this or can we use `.`?

Reply:
Oh yeah! Now TechDocs is pushing for `dir:.` in the templates.

Reply:
Let's do it.

Reply:
DO IT HIMANSHU!